docs/src/main/sphinx/connector/faker.md

```diff
@@ -102,6 +102,15 @@ The following table details all supported column properties.
     sentence from the
     [Lorem](https://javadoc.io/doc/net.datafaker/datafaker/latest/net/datafaker/providers/base/Lorem.html)
     provider.
+* - `min`
+  - Minimum constraint for generated values, inclusive. Cannot be set for
+    character-based type columns.
+* - `max`
+  - Maximum constraint for generated values, inclusive. Cannot be set for
+    character-based type columns.
+* - `options`
```

Comment:
options -> values
"Options" can be misunderstood for some kind of config

Reply:
I applied this change, but since `values` is a reserved SQL keyword, it'll always require double quotes.

Reply:
Let's use `allowedValues` to avoid the extra quotes then

Reply:
ping

Reply:
Or `allowed_values`? Properties are probably case insensitive.

Reply:
yes

Reply:
Done!